addons/docs/README.md

```diff
@@ -108,6 +108,14 @@ Docs has peer dependencies on `react`, `react-is`, and `babel-loader`. If you wa
 yarn add -D react react-is babel-loader
 ```
 
+If using Create React App, enable [this preset](https://github.com/storybookjs/presets/tree/master/packages/preset-create-react-app). Otherwise
+
+```sh
+yarn add -D babel-plugin-react-docgen
+```
+
+and add `react-docgen` to babelrc.
```

Comment:
Only if you have a custom babelrc? I believe Storybook's default already includes this.

Reply:
This is beyond my understanding. When I remove my `.babelrc`, I get syntax errors around my JSX in storybook. I would have assumed without my babelrc storybook would have used a custom config, but would that custom config automatically have `preset-react`? How can I find that custom config?

I simply followed the config [here](https://github.com/storybookjs/babel-plugin-react-docgen#usage) which I thought will always be necessary with addon-docs?